cost-analyzer/values.yaml

```diff
@@ -592,7 +592,7 @@ networkCosts:
   port: 3001
   # this daemonset can use significant resources on large clusters: https://guide.kubecost.com/hc/en-us/articles/4407595973527-Network-Traffic-Cost-Allocation
   resources:
-    limits:
+    limits: # remove the limits by setting limits: null
```

Comment:
I think the common null value here is just an empty object: `{}`  although I'll be honest, semantically, null makes way more sense :)

Reply:
fixed. agree for consistency.